mempool/reactor_test.go

```diff
@@ -434,8 +434,8 @@ func TestMempoolReactorMaxActiveOutboundConnectionsNoDuplicate(t *testing.T) {
 
 	// Now the third reactor should start receiving transactions from the first reactor and
 	// the fourth reactor from the second
-	checkTxsInOrder(t, txs, reactors[2], 0)
-	checkTxsInOrder(t, txs, reactors[3], 0)
+	checkTxsInMempool(t, txs, reactors[2], 0)
```

Comment:
The tests do not need to confirm order as we already have separate unit tests to confirm that works.